### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2237,6 +2237,30 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Cluster Log Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting defines the frequency of cluster request time logging for [performance monitoring](https://docs.mattermost.com/deployment/metrics.html), measured in milliseconds.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": "2000"`` with whole number input.                                                       |
```
Comment (JeffSchering) [inline source/administration/config-settings.rst]: If it's whole number input, then it should be ``2000`` only, without the quotes.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2237,6 +2237,30 @@ There are a number of settings customizable in ``config.json`` unavailable in th
...
+| This feature's ``config.json`` setting is ``"EnableUserTypingMessages": "true"`` with string input.                                                                  |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Time Between User Typing Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting defines how frequently "user is typing..." messages are updated, measured in milliseconds.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TimeBetweenUserTypingUpdatesMilliseconds": "5000"`` with string input.                                                  |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (JeffSchering) [inline source/administration/config-settings.rst]: Is it true that this is string input, not whole number input?

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -2237,6 +2237,30 @@ There are a number of settings customizable in ``config.json`` unavailable in th
 Service Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Cluster Log Timeout
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting defines the frequency of cluster request time logging for [performance monitoring](https://docs.mattermost.com/deployment/metrics.html), measured in milliseconds.
+
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Let me know if the relative link included is correct or not

File: source/administration/config-settings.rst (hunk 1 above)
Comment (jasonblais) [inline source/administration/config-settings.rst]: whoops, sorry. Fixed

File: source/administration/config-settings.rst (hunk 2 above)
Comment (jasonblais) [inline source/administration/config-settings.rst]: it's a whole number input..copy-paste error

**Pull request merged**